examples/asg.cpp

```diff
@@ -51,17 +51,15 @@ int main() {
   fal.addNode(true);
   for (int l = 1; l <= L; l++) {
     int label = output[l-1];
-    fal.addNode(false, l == L - 1);
+    fal.addNode(false, l == L);
```

Comment:
Thanks for catching this! Would you mind updating the comment in L50 to the following correct URL?

https://bit.ly/36to2xk

Reply:
Please also provide the URL at line L61

https://github.com/facebookresearch/gtn/blob/adaa8047cfab0f91048c336c6350565073ecb1d1/examples/asg.cpp#L61

Reply:
Here is the one for L60: https://git.io/JUF9n

Reply:
Updated.